src/scenes/home/benefit/benefit.css

```diff
@@ -52,6 +63,21 @@
   .tickets > div {
     width: 300px;
   }
+  .logo {
+    max-width: 50%;
+    height: auto;
+    margin: 2vw;
+  }
+  .srvdLogo {
+    height:auto;
+    max-width: 50%;
+    margin: 2vw;
+  }
+  .pvaLogo {
+    max-width: 75%;
+    height: auto;
+    margin: 2vw;
+  }
 }
```

Comment:
`height` and `margin` rules in this media query appear to be duplicates of the non-media-query rule, so they probably don't need to be here. Only the `max-width` rules.